ci_scripts/package.sh

```diff
@@ -9,6 +9,11 @@ mkdir -p ${CI_ARTIFACTS_DIR} || true
 
 cp ${DS_DSDIR}/tensorflow/bazel*.log ${CI_ARTIFACTS_DIR}/
 
+if [ -d ${DS_TFDIR}/bazel-bin/native_client/stt_wasm_bindings ]; then
+    package_libstt_wasm "libstt.zip"
+    exit 0
```

Comment:
It's a bit of a weird way to set it up, if this folder exists the script behaves completely differently. Can we either make it a CLI parameter or an independent script?